iris-mpc-cpu/src/hawkers/plaintext_store.rs

```diff
@@ -132,23 +98,24 @@ impl VectorStore for PlaintextStore {
         query: &Self::QueryRef,
         vector: &Self::VectorRef,
     ) -> Self::DistanceRef {
-        // Do not compute the distance yet, just forward the IDs.
-        (*query, *vector)
+        let query_code = &self.points[*query as usize];
```

Comment:
This recasts from `PointId` are a bit annoying. It might be worth to write a getter (just named `get`) for points that accepts only `PointId`.

Reply:
Since `PointId`s are intended to be used as indices, I tried implementing `Index` and `IndexMut` for `Vec` indexed by `PointId`, which seems to simplify the notation as well.  Does this approach address the issue for you?

(I also added `From<u32>` and `From<usize>` to simplify some other conversions a bit.)